environment.yml

```diff
@@ -24,17 +25,17 @@ dependencies:
 - imageio
 - tqdm
 - spyder-kernels
+- imageio-ffmpeg>=0.4.1
+- dipy
+- nibabel
+- nilearn
+- python-picard
+- pyqt>=5.10
+- vtk>=9.0.1
+- pyvista>=0.24
+- pyvistaqt
+# These are stuck in pip until mayavi updates
 - pip:
   - mne
-  - imageio-ffmpeg>=0.4.1
-  - vtk>=9.0.1
-  - pyvista>=0.24
-  - pyvistaqt
   - https://github.com/enthought/mayavi/archive/master.zip
-  - PySurfer[save_movie]
-  - dipy --only-binary dipy
-  - nibabel
-  - nilearn
-  - neo
-  - python-picard
-  - PyQt5>=5.10
+  - PySurfer
```

Comment:
This is available in `conda-forge`.

Reply:
See note in the code, it's because mayavi has not released a vtk9 compatible version

Reply:
OK, although the versions on pip and conda are identical?

Reply:
Yes, because PySurfer depends on Mayavi, if it's in the conda section it will try to pull in a bad version that segfaults.

Because the MNE recipe depends on PySurfer, same story

Reply:
(Segfaults during install)